### PR TITLE
create calculator

### PR DESCRIPTION
**Pull request by denisbielykh**

### CODE REVIEW

File: src/makeCalculator.js [hunk 1]
```diff
@@ -37,7 +37,38 @@
...
+      return result - value;
+    },
+
+    multiply(result, value) {
+      return result * value;
+    },
+
+    divide(result, value) {
+      return result / value;
+    },
```
Comment (lerastarynets) [inline src/makeCalculator.js:68]: pls add case with dividing to 0, and return an error because it's impossible

**Review by lerastarynets — CHANGES_REQUESTED** [review]
Nice start! Let's fix and improve your work a little bit)

File: src/makeCalculator.js (hunk 1 above)
Comment (denisbielykh) [reply src/makeCalculator.js]: console.log('Error'); линтер не пропускает. Добавил просто проверку на 0

**Review by etojeDenys — APPROVED** [review]
good job

**Pull request closed without merging**